src/init.jl

```diff
@@ -6,6 +6,7 @@ function __init__()
     @require MiniQhull = "978d7f02-9e05-4691-894f-ae31a51d76ca" include("Initialization/init_MiniQhull.jl")
     @require Optim = "429524aa-4258-5aef-a3af-852621145aeb" include("Initialization/init_Optim.jl")
     @require Polyhedra = "67491407-f73d-577b-9b50-8179a7c68029" include("Initialization/init_Polyhedra.jl")
+    @require StaticArrays = "67491407-f73d-577b-9b50-8179a7c68029" include("Initialization/init_StaticArrays.jl")
```

Comment:
this uuid is not correct. i can fix it together with other commit that simplifies the optional code.